string status
string message

MapDetail detail